mlflow/keras.py

```diff
@@ -1,861 +1,14 @@
-"""
-The ``mlflow.keras`` module provides an API for logging and loading Keras models. This module
-exports Keras models with the following flavors:
+import mlflow
 
-Keras (native) format
-    This is the main flavor that can be loaded back into Keras.
-:py:mod:`mlflow.pyfunc`
-    Produced for use by generic pyfunc-based deployment tools and batch inference.
-"""
-import importlib
-import os
-import re
-import yaml
-import tempfile
-import shutil
-import warnings
 
-import pandas as pd
+# Redirect `mlflow.keras._load_pyfunc` to `mlflow.tensorflow._load_pyfunc`,
+# For backwards compatibility on loading keras model saved by old mlflow versions.
+_load_pyfunc = mlflow.tensorflow._load_pyfunc
```

Comment:
from mlflow.tensorflow import _load_pyfunc

Does this work?

Reply:
This works. But I prefer to use `_load_pyfunc = ...` so it can avoid lint errors.

Reply:
You can ignore lint erros with `pylint: disable`.

Reply:
```
# pylint: disable-next=unused-import
from mlflow.tensorflow import _load_pyfunc, load_model, log_model, save_model, autolog
```